src/AspNetCoreModuleV2/RequestHandlerLib/requesthandler_config.cpp

```diff
@@ -251,6 +256,12 @@ REQUESTHANDLER_CONFIG::Populate(
         goto Finished;
     }
 
+    FINISHED_IF_FAILED(ConfigUtility::FindDebugFile(pAspNetCoreElement, debugFile));
```

Comment:
I don't love having debug initialize in config ctor.

Reply:
I don't either, but its either that or saving it in the config and then using it immediately after.

Reply:
Or you can add initialize method to debugutils that takes `IHttpApplication` and does reading there.

Reply:
Yes, it would still be done on the first request but at least outside and before actual config reading.